raiden/ui/app.py

```diff
@@ -323,6 +323,12 @@ def run_app(
             )
             sys.exit(1)
 
+        services_deployment_data = get_contracts_deployed(
+            chain_id=node_network_id,
+            version=contracts_version,
+            services=True,
+        )
+        services_contracts = services_deployment_data['contracts']
```

Comment:
AFAIK, this will fail if you run raiden in production-mode. I might be mistaken though

Reply:
You are right. This interface is ridiculous. It makes no sense. Let me fix this then.